pkg/reconciler/pipelinerun/resources/conditionresolution_test.go

```diff
@@ -290,9 +290,6 @@ func TestResolvedConditionCheck_ToTaskResourceBindings(t *testing.T) {
 
 	expected := []v1alpha1.TaskResourceBinding{{
 		Name: "git-resource",
-		ResourceRef: v1alpha1.PipelineResourceRef{
```

Comment:
Why was this deleted?

Reply:
Sometimes you want to have a ResourceBinding without a Ref (e.g. if you're using a spec!). So in the tests for embedded TaskRun Resource specs, this was being set even tho it shouldn't be

(The other function actually doesn't set this)

(Might be worth adding this to the commit message if it isn't there already @pritidesai !)